scalafmt-tests/src/test/resources/scala3/ImportingGivens.stat

```diff
@@ -0,0 +1,46 @@
+#
+<<< wildcard import
+object B {
+  import A._
+  import A.given}
+>>>
+object B {
+  import A._
+  import A.given
+}
+<<< merged import clauses
+object B {
+  import A.{given, _}
+}
+>>>
+object B {
+  import A.{given, _}
+}
+<<< import given of type
+maxColumn = 20
```

Comment:
can you add the same but with 10?

Reply:
looks same, corresponds with the situation without given